pkg/image/docker/manifest.go

```diff
@@ -70,10 +70,10 @@ func extractManifest(tarPath string) (*dockerManifest, error) {
 }
 
 // generateOCIManifest takes a docker manifest and a path to the tar and generates an OCI manifest derived from the given arguments and the docker config.
-func generateOCIManifest(tarPath string, manifest *dockerManifest) (*v1.Manifest, error) {
+func generateOCIManifest(tarPath string, manifest *dockerManifest) (*v1.Manifest, []byte, error) {
```

Comment:
We should update the doc comment to make it clear what the []byte return value is